css-layout-classes/styles.css

```diff
@@ -0,0 +1,43 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+h1 {
+  margin-left: 25%;
+  font-weight: 700;
+}
+
+h2 {
+  font-weight: 700;
+}
+
+p {
+  font-weight: 600;
+}
+
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
There shouldn't be a need for `flex-direction: column` here, this should be doable with just setting `width` on the elements using `column` classes.